### PR TITLE
Created success and error pages, moved logo header to own component

### PR DESCRIPTION
**Pull request by devinowen**
These are the pages we'll want to redirect to once the auth and playlist saving process is complete on the backend.

If the process is successful, let's redirect to `/success?venue_id=venue_id`. I _think_ I can infer the playlist URI from the ID, so that will be simplest. If for some reason I can't infer the URI from the venue ID alone we'll need to pass it instead.

If the process is unsuccessful, redirect to `/error`.

Let me know if you have any questions about how to do this in the code on your end!

### CODE REVIEW

File: webapp/src/pages/error/index.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/** @jsx jsx */
```
Comment (devinowen) [inline webapp/src/pages/error/index.tsx:1]: Screen Shot 2022-01-31 at 10 26 08 PM

File: webapp/src/pages/success/index.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+/** @jsx jsx */
```
Comment (devinowen) [inline webapp/src/pages/success/index.tsx:1]: Screen Shot 2022-01-31 at 10 26 25 PM

**Comment by samifriedrich** [comment]
Okay, sounds good. I'll implement this into the final endpoint.  And I think you meant `/success?playlist_id=playlist_id`, not venue_id, right?

**Comment by devinowen** [comment]
> Okay, sounds good. I'll implement this into the final endpoint. And I think you meant `/success?playlist_id=playlist_id`, not venue_id, right?

Whoops, I sure did! Will update that and merge this once I've confirmed that I only need `playlist_id` to generate a URI.

File: webapp/src/pages/success/index.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import { useRouter } from 'next/router'
+
+import * as styles from './Success.styles'
+import Layout from '../../components/Layout/Layout'
+import HeaderLogo from '../../components/HeaderLogo/HeaderLogo';
+import InfoPage from '../../components/InfoPage/InfoPage';
+
+const Success = () => {
+  const router = useRouter()
+  const { playlist_id: playlistId } = router.query
```
Comment (devinowen) [inline webapp/src/pages/success/index.tsx:13]: It works! Loaded a playlist from my account:

Screen Shot 2022-02-01 at 5 33 00 PM

File: webapp/src/pages/success/index.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+          </a>
+        </Link>
+
+        <p>
+          Or, listen to it directly here:
+        </p>
+
+        <iframe
+          sx={styles.playlist}
+          src={`https://open.spotify.com/embed/playlist/${playlistId}`}
```
Comment (devinowen) [inline webapp/src/pages/success/index.tsx:40]: This URL works, but for some reason about 50% of the time I'd load the page I would get a `Bad Request` message in the iframe. The other 50% was peachy keen. Let's merge for now and if the problem persists when we can troubleshoot then.

**Comment by devinowen** [comment]
Ok! This should be good to go, so I'm going to merge to keep everything up to date.

The UI is super basic as you can see, so feel free to suggest updates. I was going for speed over complex design to get it working, but I'm totally open to ideas on how to make these pages even better.

As discussed above, once everything else works, you can have the flask app redirect to `/success?playlist_id=playlist_id` on success and `/error` if something goes wrong.

**Pull request merged**